va/va.h

```diff
@@ -3193,15 +3214,17 @@ typedef  struct _VACodedBufferSegment  {
     /** \brief Reserved for future use. */
     uint32_t        reserved;
     /** \brief Pointer to the start of the data buffer. */
-    void               *buf;
+    void            *buf;
     /**
      * \brief Pointer to the next #VACodedBufferSegment element,
      * or \c NULL if there is none.
      */
-    void               *next;
-
-    /** \brief Reserved bytes for future use, must be zero */
-    uint32_t                va_reserved[VA_PADDING_LOW];
+    void            *next;
+    /** 
+      *\brief IV or Counter for encrytion filled by driver.
+      * Only available when status is set with \c VA_CODED_BUF_STATUS_VALID_IV.
+      */ 
+    uint32_t        encryption_iv[4];
```

Comment:
Any particular reason why this is uint32_t[4]?  uint8_t[16] might be clearer, and would avoid any possible endianness issues.